src/commands/source/download.rs

```diff
@@ -314,5 +314,16 @@ pub async fn download(
     }
 
     debug!("r = {:?}", r);
+
+    let verify_result = super::verify(matches, config, repo, progressbars).await;
+    match verify_result {
+        Ok(()) => {
+            info!("Verification succeeded");
+        }
+        Err(e) => {
+            return Err(e);
+        }
+    }
```

Comment:
I didn't test it but it seems a good idea to simplify this like that:
```suggestion
super::verify(matches, config, repo, progressbars).await?;
```

I don't think the info output adds any value given that there's already the normal stdout output.

Reply:
agreed